### ISSUE TITLE
Compile User Contributed Documentation If Possible

### ISSUE THREAD

**Issue by novaTopFlex** [zealdocs/zeal#1616]
### Discussed in https://github.com/orgs/zealdocs/discussions/1608

<div type='discussions-op-text'>

<sup>Originally posted by **novaTopFlex** March 23, 2024</sup>
I understand that the "docsets" are provided via Dash for macOS, but I would also like to add documentation that may or may not already be on Dash. If the appropriate documentation for Kotlin is available, could someone please add the documentation to the Zeal browser?

Why is Kotlin such an important programming language? Kotlin, along with other programming languages and accessories developed by JetBrains, is a common programming language for Google Android markets and to a lesser extent on Microsoft Windows markets. Since many programmers are expected to program on a Linux-based system, I would like to have a Kotlin docset option added to Zeal.</div>

### Background

I was initially unable to find Kotlin documentation until I had discovered a repository here on this platform of finished contributed docset feeds that required macOS for compilation. I would like to replace the current list of available docset feeds with a tree view separating the standard feeds from the feeds of user contributions. I am speaking from a Linux point of view, and would like to be able to simplify the process of installing user contributions to Zeal (as with Dash on macOS). The trouble that I had encountered was specifically when I was unable to install the user-contributed QEMU docset to my Zeal documentation browser because I was not a user of macOS or the proprietary Dash.

**Comment by trollixx** [comment]
Really a duplicate of #170. There are a few prerequisites left before we can make it happen.
